--- cz.orig	2016-06-23 21:30:04.646742027 +0200
+++ cz	2017-03-21 01:03:00.703710684 +0100
@@ -145,6 +145,46 @@
     key <AD08> { [iacute,      Iacute      ] };
     key <AD09> { [oacute,      Oacute      ] };
     key <AD10> { [odiaeresis,  Odiaeresis  ] };
+
+
+    // =========================================================
+    // ZACATEK MEHO ROZSIRENI UCW KLAVESNICE
+    // Vsechny popisky vzhledem k anglicke klavesnici!
+    // mapovani: /usr/include/xkbcommon/xkbcommon-keysyms.h
+
+    // ; => dvojite uvozovky dole ceske, : => jednoduche uvozovky dole
+    key <AC10> { [ doublelowquotemark, singlelowquotemark ] };
+    // ' => dvojite uvozovky nahore ceske, " => jednoduche uvozovky nahore
+    key <AC11> { [ leftdoublequotemark, leftsinglequotemark ] };
+    // space => nedelitelna mezera, shift+space => kratka mezera
+    key <SPCE> { [ nobreakspace, thinspace ] };
+    // . => trojtecka (vypustka)
+    key <AB09> { [ ellipsis ] };
+    // 1 => znak odstavce, shift+1 => trademark
+    key <AE01> { [ section, trademark ] };
+    // 2 => registrovana znacka, shift+2 => copyright
+    key <AE02> { [ registered, copyright ] };
+    // 3 => PI
+    key <AE03> { [ Greek_pi ] };
+    // 4 => euro
+    key <AE04> { [ EuroSign ] };
+    // 8 => znak nekonecno
+    key <AE08> { [ infinity ] };
+    // 9 => smajl :-)
+    key <AE09> { [ U1F603 ] };
+    // 0 => smajl :-(
+    key <AE10> { [ U1F623 ] };
+    // = => nerovna se, + => plus minus
+    key <AE12> { [ notequal, plusminus ] };
+    // - => kratka pomlcka (ceska "pulctvercikova"), _ => dlouha
+    key <AE11> { [ endash, emdash ] };
+    // ] => bocni uvozovky dvojite prave, } => jednoduche prave
+    key <AD11> { [ guillemotright, U203A ] }; //anglequoright
+    // [ => bocni uvozovky dvojite leve, { => jednoduche leve
+    key <AD12> { [ guillemotleft, U2039 ] }; //anglequoleft
+
+    // KONEC MEHO ROZSIRENI
+    // =========================================================
 };
 
 partial alphanumeric_keys
